src/Microsoft.DotNet.Build.Tasks.Feed/SleetLogger.cs

```diff
@@ -43,7 +43,9 @@ public void LogDebug(string data)
 
         public void LogError(string data)
         {
-            _log.LogError(data);
+            // There are cases where Sleet fails and we retry on our side causing things to actually work but if Sleet logs an error the whole build leg
+            // is marked as failed even though it actually succeeded, hence we log a warning here but we will log an error if the retry did not help
```

Comment:
I meant something should be added in the warning message to alert anyone trying to diagnose sleet errors that show up as warnings.

Reply:
These comments have been addressed in https://github.com/dotnet/buildtools/pull/1791